sa/proto/sa.proto

```diff
@@ -255,7 +255,7 @@ message NewAuthzRequest {
   // Next unused field number: 13
   reserved 1; // previously id
   reserved 2; // previously dnsName
-  Identifier identifier = 12;
+  core.Identifier identifier = 12;
```

Comment:
I think this is a deployability problem because the message type changes, right?

Reply:
Nope, all messages are the same type, "message". As long as the fields of the replacement message have the same types and numbers, the replacement is invisible on the wire.